crates/subspace-service/src/sync_from_dsn/snap_sync.rs

```diff
@@ -379,42 +414,35 @@ where
 
     debug!(info = ?client.info(), "Snap sync finished successfully");
 
-    Ok(())
+    Ok(true)
 }
 
-async fn wait_for_block_import<Block, Client>(
+pub async fn wait_for_block_import<Block, Client>(
```

Comment:
If you want to reuse this, move it to `utils` module or something, it doesn't look nice to import something from `snap_sync` internals just because you don't duplicate it somewhere else, looks a bit like dependency spaghetti to me.